shared/tracker/index.desktop.js

```diff
@@ -106,7 +106,9 @@ export default class TrackerRender extends PureComponent<RenderProps> {
                     style={{
                       ...globalStyles.flexBoxRow,
                       alignItems: 'flex-start',
+                      cursor: 'pointer',
                       justifyContent: 'flex-start',
+                      marginBottom: globalMargins.xtiny,
```

Comment:
I'm not sure if this line is what's causing it, but if you scroll down on a tracker the background color shows through briefly:
<img width="320" alt="tracker_-_mikem_and_developer_tools_-_file____users_dan_go_src_github_com_keybase_client_shared_desktop_renderer_renderer_html_mainwindow_react_perf" src="https://user-images.githubusercontent.com/11968340/36558853-6b5f0676-17da-11e8-96ea-59d584969218.png">

Doesn't appear on master.